src/arch/x86/tlb.hh

```diff
@@ -147,7 +147,7 @@ namespace X86ISA
         Fault finalizePhysical(const RequestPtr &req, ThreadContext *tc,
                                Mode mode) const override;
 
-        TlbEntry *insert(Addr vpn, const TlbEntry &entry);
+        TlbEntry *insert(Addr vpn, const TlbEntry &entry, int asn);
```

Comment:
What is "asn"?

We need to add a comment here.

Reply:
It should be pid now. Updated in a new commit